app/models/user.rb

```diff
@@ -292,8 +292,8 @@ def remember_me
   end
 
   def cached_followed_tag_names
-    cache_name = "user-#{id}-#{updated_at}/followed_tag_names"
-    Rails.cache.fetch(cache_name, expires_in: 24.hours) do
+    cache_name = "user-#{id}-#{following_tags_count}-#{last_followed_at&.strftime('%Y-%m-%d-%H:%M:%S')}/followed_tag_names"
```

Comment:
I really want to make these keys easier to find in Redis so I added some formating. I think we should probably format everything this way when possible so that we are consistent.

Reply:
ahaha I was thinking about the same thing while I was testing the cache view helper with Redis, that the date was weirdly formatted.

Can I suggest using the `.rfc3339` method? It's easy, it's consistent and it's easy to find using regexps:

```ruby
[2] pry(main)> Time.current.rfc3339
=> "2019-11-13T18:04:55Z"
```

Reply:
I have never used that before but I LOVE IT! TIL!